neurolib/models/multimodel/builder/base/network.py

```diff
@@ -137,6 +163,34 @@ def get_nested_params(self):
             nested_dict[node_key][mass_key] = mass.params
         return nested_dict
 
+    def make_params_symbolic(self, vector=True):
```

Comment:
this transforms all model parameters into `sp.Symbol` instead of their float value - needed for new `numba` backend